web/index.php

```diff
@@ -53,45 +170,80 @@
 	$handle->execute();
 	$data = $handle->fetchAll();
 	echo '<center><h1>Flightlog for '.htmlspecialchars($_GET['airfield']).'</h1>';
-	echo '<h2>';
-	echo '<a href="?airfield='.$_GET['airfield'].'&d='.$previousDate->format('Y-m-d').'">&lt;</a> ';
```

Comment:
So we are removing the ability to move in time by going forward and backward by one day?
+ using date in URL so we can copy/past to friends or directly go to this date.
Can we get back this feature?

Reply:
Complete parameter set in URL: done. I was originally struggling using the form's GET method, because I was not able to provide additional GET-values (parameters) not part of the form. Found a solution using a hidden forms element.

Removing the ability to step forward and backward one day: I find it a little redundant to have additional possibilities other than the date picker to manipulate the date. If you like this possibility very much, I can try to add it in a next version.

Last question: should I change the default DateTime format to something different than RFC? RFC contains redundant information (date) two times for every row - the date is already shown in the date-picker.

Reply:
Thanks.

For forward/backward, yes i think this is a very useful function.

About DateTime fortmat, yes we can set it by default to UTC and hh:mm maybe?